src/object.rs

```diff
@@ -566,6 +566,11 @@ pub trait ObjectExt: IsA<Object> {
     fn has_property<'a, N: Into<&'a str>>(&self, property_name: N, type_: Option<Type>) -> Result<(), BoolError>;
     fn get_property_type<'a, N: Into<&'a str>>(&self, property_name: N) -> Option<Type>;
 
+    fn block_signal(&self, handler_id: &SignalHandlerId);
+    fn unblock_signal(&self, handler_id: &SignalHandlerId);
+    fn stop_signal_emission(&self, signal_id: u32, detail: glib_ffi::GQuark);
```

Comment:
No ffi types in the API here. We need GQuark bindings, or pass &str here and convert internally

Reply:
Should I just expose `stop_signal_emission_by_name` then? Renamed to `stop_signal_emission` maybe?

Reply:
Yeah, let's do the quark stuff once we have bindings for that. That's also how I did it for the signal emitter and connector function

Reply:
I mean, also rename for now. We can break API later once quarks are there

Reply:
Done.